test/integration/integration_test.go

```diff
@@ -259,6 +261,8 @@ func verifyHTTPResponse(partOfResponse, url, urlSuffix, assertion, expected stri
 	switch url {
 	case "OpenShift":
 		url = minishift.getOpenShiftUrl() + urlSuffix
+	case "Routing":
```

Comment:
Just "Routing" is not enough, it would need to be something like "route for <app-name>"